test/lib/ufe/testEditRouting.py

```diff
@@ -125,67 +128,65 @@ def setUp(self):
         self.assertTrue(self.pluginsLoaded)
 
         cmds.file(new=True, force=True)
-        import mayaUsd_createStageWithNewLayer
- 
-        # Create the following hierarchy:
+
+    def tearDown(self):
+        # Restore default edit routers.
+        mayaUsd.lib.restoreAllDefaultEditRouters()
+
+    def _prepareSimpleScene(self):
```

Comment:
Not all tests use teh same scene anymore, so the scene setup was moved from setUp() to a helper function.